qoqo-iqm/src/backend.rs

```diff
@@ -211,36 +211,6 @@ impl BackendWrapper {
             .map_err(|err| PyRuntimeError::new_err(format!("Running Circuit failed: {:?}", err)))
     }
 
-    /// Run all circuits corresponding to one measurement with the IQM backend.
-    ///
-    /// An expectation value measurement in general involves several circuits.
-    /// Each circuit is passed to the backend and executed separately.
-    /// During execution values are written to and read from classical registers
-    /// (List[bool], List[float], List[complex]).
-    /// To produce sufficient statistics for evaluating expectation values,
-    /// circuits have to be run multiple times.
-    /// The results of each repetition are concatenated in OutputRegisters
-    /// (List[List[bool]], List[List[float]], List[List[complex]]).  
-    ///
-    /// Args:
-    ///     measurement (Measurement): The measurement that is run on the backend.
-    ///
-    /// Returns:
-    ///     Tuple[Dict[str, List[List[bool]]], Dict[str, List[List[float]]]], Dict[str, List[List[complex]]]]: The output registers written by the evaluated circuits.
-    ///
-    /// Raises:
-    ///     TypeError: Circuit argument cannot be converted to qoqo Circuit
-    ///     RuntimeError: Running Circuit failed
-    pub fn run_measurement_registers(&self, measurement: &PyAny) -> PyResult<Registers> {
```

Comment:
How come this function has been removed from the API?

Reply:
This function that I have removed would be more appropriately `run_circuit_batch`, which is basically what thw new `run_measurement` does, only the latter immediately processes the results to evaluate the measurement. Do you think it would be useful to also add a separate function `run_circuit_batch` that gives you back the raw results?

Reply:
No, this makes sense, sounds good